File: kalikesia.com/backend-software-engineer/nganuin-number/number.rb
buf = 1
(1..100).each { |i| buf = i == 1 ? i : i * buf }

bufstr = buf.to_s
buflen = buf.to_s.size
result = 0

(0...buflen).each do |i|
  result = bufstr[i].to_i if i == 0 #
  if i > 0
    split = bufstr[i].to_i
    assign = result + split
    result = assign
  end
end

puts result

File: kalikesia.com/backend-software-engineer/nganuin-number/number.py
buf = 1
for i in range(1,100):
    buf = i * buf

print ">" + str(buf) # + 0

bufstr = str(buf)
buflen = len(bufstr)
result = 0

for i in range(0, int(buflen)):
    if i == 0:
        result = i
    else:
        split = int(bufstr[i])
        assign = result + split
        result = assign

print result
